--- ocaml-csv-1.1.7/Makefile.orig	2007-05-14 18:20:10.000000000 +0200
+++ ocaml-csv-1.1.7/Makefile	2015-07-26 01:52:31.364677893 +0200
@@ -21,7 +21,7 @@
 
 OCAMLDOCFLAGS := -html -stars -sort $(OCAMLCINCS)
 
-all: csv.cma csv.cmxa example csvtool
+all: csv.cma example csvtool
 
 csv.cma: $(OBJS)
 	ocamlfind ocamlc $(OCAMLCINCS) $(OCAMLCFLAGS) -a -o $@ $^
@@ -36,8 +36,8 @@
 	ocamlfind ocamlc $^ -o $@
 	./test
 
-csvtool: csv.cmxa csvtool.ml
-	ocamlfind ocamlopt $(OCAMLOPTINCS) $(OCAMLOPTFLAGS) $(OCAMLOPTLIBS) $^ -o $@
+csvtool: csv.cma csvtool.ml
+	ocamlfind ocamlc $(OCAMLOPTINCS) $(OCAMLOPTFLAGS) $(OCAMLOPTLIBS) $^ -o $@
 
 # Common rules for building OCaml objects.
 
